plugins/database/cassandra/connection_producer_test.go

```diff
@@ -42,7 +42,7 @@ func TestSelfSignedCA(t *testing.T) {
 
 	host, cleanup := cassandra.PrepareTestContainer(t,
 		cassandra.ContainerName("cassandra"),
-		cassandra.Image("bitnami/cassandra", "latest"),
+		cassandra.Image("bitnami/cassandra", "3.11"),
```

Comment:
Looks like bitnami isn't tagging a `3.11` version? Looks like we can either do `3` or `3.11.11`: https://hub.docker.com/r/bitnami/cassandra/tags?page=1&ordering=last_updated
```suggestion
		cassandra.Image("bitnami/cassandra", "3.11.11"),
```

Reply:
Whoops, I was testing with `"3"` locally which is why it worked... Thanks for the 👀 !